### DISCUSSION TITLE
Auto-complete

### DISCUSSION THREAD

**Discussion by hokutonanto** [TriliumNext/Trilium#1553] [Q&A]
I'm not sure there's an auto-complete function, I haven't seen it in the options.
Basically, I'd like to type for instance ( and have Trillium to complete it for me with the closing parenthesis )

**Comment by sigaloid** [comment]
If this implemented, maybe if the current note type is code, auto-add ending quotes as well (" -> ")

**Reply by hokutonanto** [reply]
Agree, and actually, my idea is for regular text note as well.

**Comment by zadam** [comment]
Hi, this is not supported. There would have to be support in the editing component, see https://github.com/ckeditor/ckeditor5/issues/5798
